x-pack/plugins/reporting/server/export_types/csv_from_savedobject/server/lib/generate_csv_search.ts

```diff
@@ -156,7 +156,7 @@ export async function generateCsvSearch(
   };
 
   const config = reporting.getConfig();
-  const elasticsearch = await reporting.getElasticsearchService();
+  const elasticsearch = reporting.getElasticsearchService();
```

Comment:
I'm considering deleting this helper method entirely since the dep can come from `getPluginSetupDeps`

Reply:
++ on always decreasing API surface area

Reply:
I will leave it in for now, since it allows us to mock the elasticsearch service pretty well.